src/Parser/Warrior/Fury/Modules/BuffDebuff/EnrageUptime.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import StatisticBox from 'Main/StatisticBox';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class enrageUptime extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  suggestions(when) {
+    const enrageUptime = this.combatants.selected.getBuffUptime(SPELLS.ENRAGE.id) / this.owner.fightDuration;
```

Comment:
I recommend making this a getter so it can be used from other modules (also this makes sense when you make a `suggestionThresholds` getter.